app/decorators/user_decorator.rb

```diff
@@ -2,7 +2,7 @@
 
 UserDecorator = Struct.new(:user) do
   def lockout_time_remaining
-    (Devise.allowed_otp_drift_seconds - (Time.zone.now - user.second_factor_locked_at)).to_i
+    (Devise.direct_otp_valid_for - (Time.zone.now - user.second_factor_locked_at)).to_i
```

Comment:
I agree that we should find alternatives to lockouts. I think this gem will help: https://github.com/kickstarter/rack-attack. We can discuss later.